### DISCUSSION TITLE
LED On Press, Toggle

### DISCUSSION THREAD

**Discussion by Shiro555** [cpyarger/obs-midi#151] [Ideas]
Hello guys, 

First, thank you for this wonderful plugin!

I would like to know if you are planning to integrate some sort of LED activation whether it's on press or toggle. I'm using the launchpad mk1 and for now it's working but I have no LED feedback, so I'll be using a program called Control Cast or maybe something else that helps me fake the LED activation.

Thank you.

**Comment by cpyarger** [comment]
Currently this is slated to e a part of the new device feedback system.

**Reply by pgScorpio** [reply]
When can we expect led feedback on the akai APC mini ???
Any feedback on mutes and scenes would be highly appreciated.
